src/symbolize/mod.rs

```diff
@@ -313,6 +316,42 @@ pub struct Sym<'src> {
 }
 
 
+/// The reason why symbolization failed.
+///
+/// The reason is generally only meant as a hint. Reasons reported may change
+/// over time and, hence, should not be relied upon for the correctness of the
+/// application.
+#[derive(Clone, Debug, PartialEq)]
+#[non_exhaustive]
+pub enum Reason {
+    /// The absolute address was not found in the corresponding process' virtual
+    /// memory map.
+    NotInProcessMap,
```

Comment:
naming nit, maybe more succinct "Unmapped"?

Reply:
Sure, thanks for the suggestion.